assignment4.txt

```diff
@@ -0,0 +1,9 @@
+Sean Kee
+
+1. The difference between a string and a character array is that a string is an array of a collection of characters. A string is like a book, and the character array is like the pages.
```

Comment:
sort of! they're essentially the same except a string's char array must have a null terminator ("\0")